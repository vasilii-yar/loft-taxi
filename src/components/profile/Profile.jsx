import React from 'react';
import NavBar from "../../components/navbar/NavBar";
import CreditCard from "../../components/profile/CreditCard";
import {Box} from "@material-ui/core";

export default class Profile extends React.Component {
    render() {
        return (
            <>
                <header>
                    <NavBar/>
                </header>
                <main>
                    <section>
                        <Box>
                            <CreditCard/>
                        </Box>
                    </section>
                </main>
            </>
        );
    }
}