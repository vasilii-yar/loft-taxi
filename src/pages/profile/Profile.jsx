import React from 'react';
import NavBar from "../../components/navbar/NavBar";

export default class Profile extends React.Component {
    render() {
        return (
            <>
                <header>
                    <NavBar navigateTo={this.props.navigateTo}/>
                </header>
                <main>
                    <section>

                    </section>
                </main>
            </>
        );
    }
}