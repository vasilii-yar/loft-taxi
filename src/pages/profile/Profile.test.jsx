import React from 'react';
import {render} from "@testing-library/react";
import {Profile} from "./Profile";

describe("Profile", () => {
    it("renders correctly", () => {
        const {container} = render(<Profile/>);
        expect(container.innerHTML).toMatch("Профиль");
    })
});