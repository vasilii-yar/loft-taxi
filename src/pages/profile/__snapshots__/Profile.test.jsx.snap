// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile renders correctly check by snapshot 1`] = `
Array [
  <header>
    <div>
      Профиль
    </div>
  </header>,
  <main>
    <section />
  </main>,
]
`;
